cli/src/pixl_cli/_database.py

```diff
@@ -114,11 +114,15 @@ def _add_new_image_to_session(extract: Extract, message: Message, session: Sessi
     return new_image
 
 
-def images_for_project(project_slug: str) -> list[Image]:
+def processed_images_for_project(project_slug: str) -> list[Image]:
```

Comment:
would be good to have a test which ensures that this works as we'd expect (and stops us reverting this behaviour)